rpc/wrtc_server_stream.go

```diff
@@ -244,8 +259,7 @@ func (s *webrtcServerStream) onRequest(request *webrtcpb.Request) {
 }
 
 func (s *webrtcServerStream) processHeaders(headers *webrtcpb.RequestHeaders) {
-	s.logger = utils.AddFieldsToLogger(s.logger, "method", headers.GetMethod())
-	s.logger.Debug("incoming grpc request")
+	s.logger.Debugw("incoming grpc request msg", "msg", s.msgCount)
```

Comment:
I don't think we should add these logs, maybe if we find a use case later on? but right now its just going to make debug output even more noisy without too much additional value @benjirewis